mongo/integration/unified_spec_test.go

```diff
@@ -229,11 +239,8 @@ func runSpecTestCase(mt *mtest.T, test *testCase, testFile testFile) {
 		if len(test.SkipReason) > 0 {
 			mt.Skip(test.SkipReason)
 		}
-		if test.Description == cseMaxVersionTest {
-			// This test checks to see if the correct error is thrown when auto encrypting with a server < 4.2.
-			// Currently, the test will fail because a server < 4.2 wouldn't have mongocryptd, so Client construction
-			// would fail with a mongocryptd spawn error. SPEC-1403 tracks the work to fix this.
-			mt.Skip("skipping maxWireVersion test")
+		if _, ok := skippedTestDescriptions[test.Description]; ok {
+			mt.Skip("skipping due to known failure")
```

Comment:
If it makes sense to make `skippedTestDescriptions` a map to the skip reasons, this can be updated to print the reason.